tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1143,21 +1112,24 @@ module.exports = {
         return utils.saveDoc(user);
       })
       .then(done, done);
+    deleteReport(done);
```

Comment:
This looks like it could call `done` twice.  It would also be neater for `teardown` to return a `Promise`, and therefore not use `done` at all, if possible.